### ISSUE TITLE
time performance  details?

### ISSUE THREAD

**Issue by BuaaAlban** [theblackcat102/Online-Speech-Recognition#1]
Hi,  would  you please  give some details about how you tested the Avg time for RNN-T?  like what does the Avg mean? etc
Thanks 

Avg Encoding Time | Avg Decoding Time | Avg Joint Time

**Comment by theblackcat102** [comment]
The encoding, decoding and joint is based on the RNNT architecture. You can reference to this figure from [Google blog](https://ai.googleblog.com/2019/03/an-all-neural-on-device-speech.html). Note we refer the predict part as decoding in the readme, they are both refering to the same part.

![image](https://user-images.githubusercontent.com/13172147/96766012-05d66880-140d-11eb-99d9-a4d558f6b02e.png)
)

Basically the average time to finish for each section (encoder, decoder, joint ) from the architecture
